src/studio-home/__mocks__/studioHomeMock.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
     },
   ],
   librariesEnabled: true,
-  libraryAuthoringMfeUrl: 'http://localhost:3001',
+  libraryAuthoringMfeUrl: 'http://localhost:3001/',
```

Comment:
The platform default for this URL [does not contain the trailing slash](https://github.com/openedx/edx-platform/blob/51ebabfd96b1d3c0d096c46981cc09e17ef11388/cms/envs/devstack.py#L178), but usually with a fully-deployed MFE, there's a `PUBLIC_PATH` variable that can default to `/`.

But I don't know for sure that our MFE configurations are consistent enough to rely on the presence of a trailing slash. Instead of constructing Library Authoring URLs from simple string concatenation, could you create a function that handles inserting the trailing slash if needed?

Reply:
Interesting, I wasn't familiar with that! That's a good idea, I've added the function here https://github.com/openedx/frontend-app-course-authoring/pull/1050/commits/567dcb48f08d7bb43ea389bda9ee3dd9c1c22966